modulefiles/module_base.gaea.lua

```diff
@@ -0,0 +1,52 @@
+help([[
+Load environment to run GFS on Gaea
+]])
+
+local spack_mod_path=(os.getenv("spack_mod_path") or "None")
+local spack_gsi_mod_path=(os.getenv("spack_gsi_mod_path") or "None")
+prepend_path("MODULEPATH", spack_mod_path)
+prepend_path("MODULEPATH", spack_gsi_mod_path)
```

Comment:
The gsi_mod_path contains the other as well.  I don't think the previous is needed.  Please confirm.